services/readme.md

```diff
@@ -0,0 +1,13 @@
+## Supported services
+The following services are currently supported by ACK. For information on how to install and use a service controller, see the [documentation](https://aws.github.io/aws-controllers-k8s/user-docs/install).
+
+|AWS Service|Status|Controller|AWS Documentation|
+|-----------|------|----------|-----------------|
+|[DynamoDB](https://aws.amazon.com/dynamodb/)|Preview|`dynamodb`|[docs link](https://docs.aws.amazon.com/dynamodb)|
+|[ElastiCache](https://aws.amazon.com/elasticache/)|Preview|`elasticache`|[docs link](https://docs.aws.amazon.com/elasticache)|
+|[SQS](https://aws.amazon.com/sqs/)|Preview|`sqs`|[docs link](https://docs.aws.amazon.com/sqs)|
+|[SNS](https://aws.amazon.com/sns/)|Preview|`sns`|[docs link](https://docs.aws.amazon.com/sns)|
+
+
+### Adding a service controller
+See our [contributors guide](/CONTRIBUTING.md) for information on adding a new service controller.
```

Comment:
Probably want to keep content to a single place. The README.md files in source code directories like `services/` should be contributor-focused documentation. Perhaps just add a link to the `docs/contents/services.md` file from here instead of repeating the content.

Reply:
fixed